.changeset/seven-oranges-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/vertexai': patch
```

Comment:
@hsubox76 Don't we need to add `'firebase': patch` here?

Reply:
No, changesets automatically does patch bumps based on dependencies but won't do a minor or major without explicit permission.